src/mapbox/mapbox.js

```diff
@@ -278,10 +281,20 @@ export default class Mapbox {
     }
   }
 
+  // If canvas size has changed, reads out the new size and returns true
+  _checkForContainerSizeChange() {
+    const {canvas} = this;
+    if (canvas && (this.width !== canvas.clientWidth || this.height !== canvas.clientHeight)) {
+      this.width = canvas.clientWidth;
+      this.height = canvas.clientHeight;
```

Comment:
How does this interact with the map controls, which still need `width` and `height` in pixels?

Reply:
Perhaps it could be handled the same way it is in deck.gl, by calling setProps on those controls?